lib/assets/blog-stylesheet.scss

```diff
@@ -1,16 +1,17 @@
-$title-font: 'Fira Sans'!default;
 $body-font: 'Merriweather' !default;
+$title-font: 'Fira Sans' !default;
 $context-font: 'Fira Sans' !default;
 $support-font: 'Fira Sans' !default;
 $body-font-composite: 'Roboto' !default;
+$title-font-composite: 'Roboto' !default;
 
 $primary-light-color: #5e7790 !default;
 $primary-dark-color: #1b242e !default;
 $accent-light-color: #ff6835 !default;
 $accent-dark-color: #cc532b !default;
 $background-light-color: #e9e9e3 !default;
 
-$black60: lighten(black, 60);
+$black60: transparentize(black, 1 - 0.60);
```

Comment:
Yeah, you're right now that I think about it; the specs do say to lower the opacity, but I don't really see why it's necessary for these article pages. We can go with `lighten()`. But, it's used the wrong way originally, I'll change it to `lighten(black, 100 - 60)`